### PR TITLE
We currently don't support unit testing models that use the `materialized view` materialization

### PR DESCRIPTION
**Pull request by dbeatty10**
[Preview](https://docs-getdbt-com-git-dbeatty10-patch-1-dbt-labs.vercel.app/docs/build/unit-tests#before-you-begin)

## What are you changing in this pull request and why?

We currently don't support unit testing models that use the [`materialized view`](https://docs.getdbt.com/docs/build/materializations#materialized-view) materialization=

See https://github.com/dbt-labs/dbt-adapters/issues/671

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

### CODE REVIEW

**Review by graciegoheen — APPROVED** [review]
thank you doug

File: website/docs/docs/build/unit-tests.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ With dbt Core v1.8 and dbt Cloud environments that have gone versionless by sele
 
 - We currently only support unit testing SQL models.
 - We currently only support adding unit tests to models in your _current_ project.
+- We currently *don't* support unit testing models that use the [`materialized view`](/docs/build/materializations#materialized-view) materialization.
```
Comment (mirnawong1) [inline website/docs/docs/build/unit-tests.md]:
```suggestion
- We currently _don't_ support unit testing models that use the [`materialized view`](/docs/build/materializations#materialized-view) materialization.
```

**Review by mirnawong1 — APPROVED** [review]
hey @dbeatty10 this looks good to me, thanks for opening! i noticed the core issue you linked was for `dbt-postres` so does this limitation apply to postgres only? or all adapters?

File: website/docs/docs/build/unit-tests.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ With dbt Core v1.8 and dbt Cloud environments that have gone versionless by sele
 
 - We currently only support unit testing SQL models.
 - We currently only support adding unit tests to models in your _current_ project.
+- We currently _don't_ support unit testing models that use the [`materialized view`](/docs/build/materializations#materialized-view) materialization.
 - We currently *don't* support unit testing models that use recursive SQL.
```
Comment (dbeatty10) [inline website/docs/docs/build/unit-tests.md]:
```suggestion
- We currently _don't_ support unit testing models that use recursive SQL.
```

**Comment by dbeatty10** [comment]
> i noticed the core issue you linked was for `dbt-postres` so does this limitation apply to postgres only? or all adapters?

I didn't confirm myself, but my understanding is that this applies to all adapters.

Internal Slack convo [here](https://dbt-labs.slack.com/archives/C03SK9W08BC/p1724854379862259).

**Review by mirnawong1 — APPROVED** [review]
lgtm! thanks @dbeatty10 !

**Review by matthewshaver — APPROVED** [review]
Thank you @dbeatty10 for always being amazing!

**Pull request merged**